pkg/restore/restore.go

```diff
@@ -1097,10 +1097,11 @@ func (ctx *restoreContext) restoreItem(obj *unstructured.Unstructured, groupReso
 	itemExists := false
 	resourceID := getResourceID(groupResource, namespace, obj.GetName())
 	resourceKind := obj.GetKind()
+	backupResourceName := obj.GetName()
 
 	restoreLogger := ctx.log.WithFields(logrus.Fields{
 		"namespace":     obj.GetNamespace(),
-		"name":          obj.GetName(),
+		"name":          backupResourceName,
```

Comment:
Let's change the field name as `original name` to better demonstrate the new and old names in the log.

Reply:
Modified.